code/modules/plumbing/plumbers/plumbing.dm

```diff
@@ -0,0 +1,102 @@
+///basic plumbing machinery it holds a wrench_act used in all plumbing machinery
+/obj/machinery/plumbing
+	name = "pipe thing"
+	icon = 'icons/obj/plumbing/plumbers.dmi'
+	icon_state = "pump"
+	anchored = FALSE
+	active_power_usage = 30
+	use_power = ACTIVE_POWER_USE
+	///how many chems can it hold
+	var/capacity = 100
```

Comment:
This makes me reeeee internally. You only call create_reagents on subtypes, yet you still have this variable here. Either move all initialization variables over here (reagent flags, capacity, plumbing component subtype) and call create_reagents and AddComponent in the Initialize for this OR remove the variable altogether. Currently it's only useful for mapedits.

Reply:
there is no point on having initialization and components here because they are all different in the subtypes

Reply:
That's exactly what I'm saying. There's no point in having this variable here.